Userland/Libraries/LibWeb/Layout/TextNode.cpp

```diff
@@ -488,62 +488,76 @@ Optional<TextNode::Chunk> TextNode::ChunkIterator::next_without_peek()
     Gfx::Font const& font = m_font_cascade_list.font_for_code_point(code_point);
     auto text_type = text_type_for_code_point(code_point);
 
+    auto broken_on_tab = false;
+
     while (m_current_index < m_utf8_view.byte_length()) {
-        code_point = current_code_point();
+        auto code_point = current_code_point();
```

Comment:
there is no need to redeclare `code_point` variable inside while loop. you could remove `auto` here.